src/helpers/load-config.py

```diff
@@ -82,6 +84,8 @@ def run():
         file = args['file']
     elif args['rollback']:
         file = f'/opt/vyatta/etc/config/archive/config.boot.{args["rollback"]}.gz'
+    elif args['saved']:
+        file = '/config/config.boot'
```

Comment:
I think best practices are to reference as:
`/opt/vyatta/etc/config/config.boot`
Though it is not an issue in this case, the lesson of https://vyos.dev/T5520 urges caution.

Reply:
Changed referenct to not use bind'ed file system